basics/indexing/text-search-support.md

```diff
@@ -212,53 +206,53 @@ Similar to other indexes, users can enable text index on a column through table
+Each column that has a text index should also be specified as `noDictionaryColumns` in `tableIndexConfig`:
 
-```javascript
+```json
 "tableIndexConfig": {
    "noDictionaryColumns": [
      "text_col_1",
      "text_col_2"
  ]}
 ```
 
-The above mechanism can be used to configure text indexes in the following scenarios:
+You can configure text indexes when:
 
 * Adding a new table with text index enabled on one or more columns.
 * Adding a new column with text index enabled to an existing table.
-* Enabling text index on an existing column.
+* Enabling a text index on an existing column.
 
 {% hint style="info" %}
-When you're using a Text index, we would recommend that you add the indexed column to the `noDictionaryColumns` columns list to reduce unnecessary storage overhead.&#x20;
+When you're using a text index, you should add the indexed column to the `noDictionaryColumns` columns list to reduce unnecessary storage overhead.
 
-For instructions on that config property, see the [Raw value forward index](forward-index.md#raw-value-forward-index) documentation.
+For instructions on that configuration property, see the [Raw value forward index](forward-index.md#raw-value-forward-index) documentation.
 {% endhint %}
 
-## Text Index Creation
+## Text index creation
 
-Once the text index is enabled on one or more columns through table config, our segment generation code will pick up the config and automatically create text index (per column). This is exactly how other indexes in Pinot are created.
+Once the text index is enabled on one or more columns through a table configuration, segment generation code will automatically create the text index (per column).
 
 Text index is supported for both offline and real-time segments.
 
 ### Text parsing and tokenization
 
-The original text document (a value in the column with text index enabled) is parsed, tokenized and individual "indexable" terms are extracted. These terms are inserted into the index.
+The original text document (denoted by a value in the column that has text index enabled) is parsed, tokenized and individual "indexable" terms are extracted. These terms are inserted into the index.
 
-Pinot's text index is built on top of Lucene. Lucene's **standard english text tokenizer** generally works well for most classes of text. We might want to build custom text parser and tokenizer to suit particular user requirements. Accordingly, we can make this configurable for the user to specify on per column text index basis.
+Pinot's text index is built on top of Lucene. Lucene's **standard english text tokenizer** generally works well for most classes of text. Since it can be desirable to build a custom text parser and tokenizer to suit particular user requirements this can be made configurable for the user to specify on per column text index basis.
```

Comment:
```suggestion
Pinot's text index is built on top of Lucene. Lucene's **standard english text tokenizer** generally works well for most classes of text. To build a custom text parser and tokenizer to suit particular requirements, this can be made configurable for the user to specify on per column text index basis.
```

Reply:
Curious what can be made configurable and/or what the user can specify per column?
not clear what is a text index basis....